### ISSUE TITLE
Tests fail with ActiveTriples 0.6.1

### ISSUE THREAD

**Issue by jcoyne** [no-reply/linked_vocabs#1]
```
  1) LinkedVocabs::Controlled#load_vocabularies should load data
     Failure/Error: expect(subject.new('Image').has_subject?(RDF::URI('http://purl.org/dc/dcmitype/Image'))).to be_true
       expected true to respond to `true?`
     # ./spec/controlled_spec.rb:85:in `block (3 levels) in <top (required)>'

  2) LinkedVocabs::Validators::PropertyValidator with value in vocabulary is valid
     Failure/Error: property :type, :predicate => RDF::DC.type, :class_name => DummyAuthority
     ArgumentError:
       type is a keyword and not an acceptable property name.
     # ./spec/property_validator_spec.rb:9:in `<class:DummyAuthority>'
     # ./spec/property_validator_spec.rb:5:in `block (2 levels) in <top (required)>'

  3) LinkedVocabs::Validators::PropertyValidator with value in vocabulary is invalid with other invalid values
     Failure/Error: property :type, :predicate => RDF::DC.type, :class_name => DummyAuthority
     ArgumentError:
       type is a keyword and not an acceptable property name.
     # ./spec/property_validator_spec.rb:9:in `<class:DummyAuthority>'
     # ./spec/property_validator_spec.rb:5:in `block (2 levels) in <top (required)>'

  4) LinkedVocabs::Validators::PropertyValidator with value out of vocabulary is invalid
     Failure/Error: property :type, :predicate => RDF::DC.type, :class_name => DummyAuthority
     ArgumentError:
       type is a keyword and not an acceptable property name.
     # ./spec/property_validator_spec.rb:9:in `<class:DummyAuthority>'
     # ./spec/property_validator_spec.rb:5:in `block (2 levels) in <top (required)>'

  5) LinkedVocabs::Validators::PropertyValidator with value of wrong class is invalid
     Failure/Error: property :type, :predicate => RDF::DC.type, :class_name => DummyAuthority
     ArgumentError:
       type is a keyword and not an acceptable property name.
     # ./spec/property_validator_spec.rb:9:in `<class:DummyAuthority>'
     # ./spec/property_validator_spec.rb:5:in `block (2 levels) in <top (required)>'

  6) LinkedVocabs::Validators::PropertyValidator with literal value is invalid
     Failure/Error: property :type, :predicate => RDF::DC.type, :class_name => DummyAuthority
     ArgumentError:
       type is a keyword and not an acceptable property name.
     # ./spec/property_validator_spec.rb:9:in `<class:DummyAuthority>'
     # ./spec/property_validator_spec.rb:5:in `block (2 levels) in <top (required)>'
```

**Comment by jcoyne** [comment]
It looks like `type` is already declared here: https://github.com/ActiveTriples/ActiveTriples/blob/master/lib/active_triples/resource.rb#L191

**Comment by jcoyne** [comment]
@no-reply I can fix the tests by changing to a different term, but I'm wondering if you need it fixed specifically for `RDF.type`.

**Comment by terrellt** [comment]
@jcoyne We talked about this, but to document - :type the property was just a bad selection for the test suite.

**Comment by no-reply** [comment]
Is this happening in `0.2.0`.  I released that without seeing this, and at about the same time. I think it fixes your problem.

**Comment by jcoyne** [comment]
Fixed by 472f7ab98d0ce6cba04a0c2e451e5e7fe5904b43
